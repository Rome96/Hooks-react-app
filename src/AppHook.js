import React from 'react'

const AppHook = () => {
  return (
    <div>
      <h3>Hola gente</h3>
    </div>
  )
}
export default AppHook
