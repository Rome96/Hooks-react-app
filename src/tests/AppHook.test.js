import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import AppHook from '../AppHook';

describe("Testing component <AppHook/> ", () => {

  test("return component <AppHook/> Snapshot", () => {
    const wrapper = shallow(<AppHook/>);
    expect(wrapper).toMatchSnapshot();
  });

});
