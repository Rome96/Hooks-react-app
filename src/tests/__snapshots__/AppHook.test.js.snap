// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component <AppHook/>  return component <AppHook/> Snapshot 1`] = `
<div>
  <h3>
    Hola gente
  </h3>
</div>
`;
